deploy.sh

```diff
@@ -1,38 +1,45 @@
 #!/bin/sh
 
 # Scripts to deploy application to staging server
-# @todo #45:30min Continue implementing deploy scripts. Find out user 
-#  and domain of the staging server and replace scp and ssh with correct data.
-#  Add scripts in cron (like the one created in #47). Verify the web 
-#  application is running (execute couple fo curl requests).
+# @todo #126:30min Finish implementing deploy scripts. Add scripts in cron 
+#  (like the one created in #47). Verify the web application is running - 
+#  execute couple of curl requests.
 
 echo "-- Run tests"
```

Comment:
@bkuzmic As I understand, these script does not install the complete environment (python, postgres and so on ) and that is fine, but I would add a puzzle to describe the installation process for the staging/production servers.

Reply:
@bkuzmic In the same puzzle, explain how to call rultor to deploy 

Ex.

`@rultor deploy 0.0.1 staging`